cudax/include/cuda/experimental/__stf/internal/dot.cuh

```diff
@@ -51,25 +53,26 @@
 #include <stack>
 #include <unordered_set>
 
-/**
- * @file
- * @brief Generation of the DOT file to visualize task DAGs
- */
+//! \file
+//!
+//! \brief Generation of the DOT file to visualize task DAGs
 
 namespace cuda::experimental::stf::reserved
 {
 
-/**
- * @brief Some helper type
- */
+int get_next_prereq_unique_id();
+
+//! Sets of integer pairs used to represent edges
 using IntPairSet = ::std::unordered_set<::std::pair<int, int>, cuda::experimental::stf::hash<::std::pair<int, int>>>;
 
 class dot;
 
-// edge represent dependencies, but we sometimes want to visualize differently
-// dependencies which are related to actual task dependencies, and "internal"
-// dependencies between asynchronous operations (eg. a task depends on an
-// allocation) which are not necessarily useful to visualize.
+//! Type of the edge
+//!
+//! Edges represent dependencies, but we sometimes want to visualize
+//! differently dependencies which are related to actual task dependencies,
```

Comment:
As a non english speaking person I tripped on this. 

is it `visualize differently meow` or `differently visualize meow`?